spec/models/rounds_spec.rb

```diff
@@ -18,23 +18,127 @@
   # end
 
   describe "methods" do
-    before(:each) do
-      test_data
-    end
 
-    xit "can turnover" do
-      expect(@round.status).to eq("open")
-      @round1.close_round
-      expect(@round.reload.status).to eq("closed")
+    it "can turnover", :vcr do
+      date = Date.yesterday.strftime('%F')
+      @lat1 = "-15.74" ; @lon1 = "-54.34"
+      @lat2 = "36.53" ; @lon2 = "-116.93"
+      @lat3 = "39.74" ; @lon3 = "-104.99"
+      wf = WeatherFacade.new
+      lat11 = "62.04" ; lon11 = "129.74" ; data11 = wf.weather_data(lat11, lon11, date)
```

Comment:
I know this is just test data, but it made me realize, did we want to normalize how it is displayed (if possible on the FE), currently it is floating to I don't even know how many decimals.